src/controllers/trust.interrupt.controller.ts

```diff
@@ -40,10 +42,13 @@ const post = (
     logger.debugRequest(req, `${req.method} ${req.route.path}`);
 
     const firstTrustId = "1";
-    return safeRedirect(res, `${config.TRUST_ENTRY_URL + "/" + firstTrustId}`);
+    let trustDetailsUrl = config.TRUST_ENTRY_URL;
```

Comment:
Would consider calling this variable `trustEntryUrl`, to reflect the value that gets assigned.

Reply:
Changed the name as suggested.